### PR TITLE
71129 payment details

### PR DESCRIPTION
**Pull request by seanciv**

### CODE REVIEW

File: src/EA.Weee.Web/Areas/Admin/Views/ProducerSubmission/AddPaymentDetails.cshtml [hunk 1]
```diff
@@ -10,4 +15,80 @@
             @title
         </h1>
     </header>
-}
\ No newline at end of file
+
```
Comment (will-hope98) [inline src/EA.Weee.Web/Areas/Admin/Views/ProducerSubmission/AddPaymentDetails.cshtml:18]: Not sure if Im missing it but I cant see a back button on here

File: src/EA.Weee.Web/Areas/Admin/ViewModels/Producers/PaymentDetailsViewModel.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    {
+        public Guid DirectProducerSubmissionId { get; set; }
+
+        [Required]
+        [DisplayName("Payment method")]
+        public string PaymentMethod { get; set; }
+
+        [Required]
+        [DisplayName("Payment recieved date")]
+        public DateTimeInput PaymentRecievedDate { get; set; }
```
Comment (will-hope98) [inline src/EA.Weee.Web/Areas/Admin/ViewModels/Producers/PaymentDetailsViewModel.cs:18]: misspelling here, should be "PaymentReceivedDate"

File: src/EA.Weee.RequestHandlers/Admin/AddPaymentDetailsHandler.cs [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+            IUserContext userContext,
+            WeeeContext context)
+        {
+            this.authorization = authorization;
+            this.dataAccess = genericDataAccess;
+            this.userContext = userContext;
+            this.context = context;
+        }
+
+        public async Task<ManualPaymentResult> HandleAsync(Requests.Admin.AddPaymentDetails request)
```
Comment (will-hope98) [inline src/EA.Weee.RequestHandlers/Admin/AddPaymentDetailsHandler.cs:35]: Should there be tests for this?

Comment (seanciv) [reply src/EA.Weee.RequestHandlers/Admin/AddPaymentDetailsHandler.cs:35]: src/EA.Weee.RequestHandlers.Tests.Unit/Admin/AddPaymentDetailsHandlerTests.cs @will-hope98

**Pull request merged**